api-admin/src/main/java/bio/terra/pearl/api/admin/service/siteContent/SiteContentExtService.java

```diff
@@ -31,6 +31,14 @@ public Optional<SiteContent> get(
     return Optional.empty();
   }
 
+  public SiteContent create(
+      String portalShortcode, String stableId, SiteContent siteContent, AdminUser user) {
+    Portal portal = authUtilService.authUserToPortal(user, portalShortcode);
+    siteContent.setPortalId(portal.getId());
+    siteContent.setStableId(stableId);
```

Comment:
it's possible, but overwriting it is fine.  the way to interpret this endpoint is "make a new version of this stableId, using the passed-in stieContent". So it might be that someone copied a sitecontent from another portal that has a different stableId, and then they're asking to save it as a new stableId (and it might have a different portalId too).   

Throughout this application, API endpoints are generally structured so that stableIds and portalShortcodes are on the path, so that it's easier to trace what's happening via access logs. This has the drawback here (and other places) of redundant fields in payloads.